mantidproject/mantid
# Manage User Directory for workbench

**NOTE: This contains a merge of an older state of branch #21566 please ensure that #21584 is merged before this**

This PR adds the manager user directory to the workbench. It is now available from `File > Manage User Directories`. This implementation does the same as MantidPlot where multiple clicks on the `Manage User Directories` shows additional dialogs.

**Question for tester**
* Clicking on`Manage User Directories` creates additional dialog boxes. Do we want this behaviour or would it be better to refocus on the currently opened dialog box (if it is already open).

**To test:**

* Open workbench
* Open the `Manage User Directory` Dialog with through `File > Manage User Directories`
* Check it functions as expected (same as MantidPlot)

Fixes #21592

**Release Notes** 
*Does not need to be in the release notes.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
